modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/Header/index.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayAlert from '@clayui/alert';
+
+import classNames from 'classnames';
+import {useMemo, useState} from 'react';
+import {Button, ButtonDropDown} from '../../../../../common/components';
+import {ALERT_DOWNLOAD_TYPE} from '../../../utils/constants/alertDownloadType';
+import {AUTO_CLOSE_DOWNLOAD_ALERT_TIME} from '../../../utils/constants/autoCloseDownloadAlertTime';
+import {DOWNLOADABLE_LICENSE_KEYS} from '../utils/constants';
+import {ALERT_ACTIVATION_AGGREGATED_KEYS_DOWNLOAD_TEXT} from '../utils/constants/alertAggregateKeysDownloadText';
+import {getActivationKeyDownload} from '../utils/getActivationKeyDownload';
+import {getActivationKeysActionsItems} from '../utils/getActivationKeysActionsItems';
+import {getActivationKeysDownloadItems} from '../utils/getActivationKeysDownloadItems';
```

Comment:
Can we export all these functions from one index and import all at once here? I think it would be more organized to import from just one place and use destructuring like this: {getActivationKeyDownload, getActivationKeysActionsItems, getActivationKeysDownloadItems} from ¨../utils¨

Reply:
Indeed! I'm just following the others utils folders patterns that do not create a index in functions (but create in constants). If we change to that we need to change for everything. What do you think @jairmedeiros ?

Reply:
Oh my gosh. It's a good suggestion, let's do it!!